modules/apps/content-dashboard/content-dashboard-web/src/test/java/com/liferay/content/dashboard/web/internal/item/BlogsEntryContentDashboardItemTest.java

```diff
@@ -396,7 +395,7 @@ private ContentDashboardItemAction _getContentDashboardItemAction(
 
 		Mockito.when(
 			contentDashboardItemActionProvider.isShow(
-				Mockito.any(JournalArticle.class),
+				Mockito.any(BlogsEntry.class),
```

Comment:
@JanisZhang Why you need to fix these tests? Is the behavior of API method changed after upgrade?

FYI @dantewang

Reply:
Hi @tinatian,

Yes, the behaviour of API method changed.
In Mockito v1 (our version),  There is no type checking in any(...) - [source](http://static.javadoc.io/org.mockito/mockito-core/1.10.19/org/mockito/Matchers.html#any(java.lang.Class)):
> This method doesn't do any type checks, it is only there to avoid casting in your code. This might however change (type checks could be added) in a future major release.

Since Mockito 2.1.0 this method performs a **type check** -[source](https://www.javadoc.io/doc/org.mockito/mockito-core/2.1.0/org/mockito/ArgumentMatchers.html#any(java.lang.Class)) thus i changed to the type of parameter passed in.